lexers/todotxt.lua

```diff
@@ -0,0 +1,141 @@
+-- Copyright 2025 Chris Clark and Mitchell. See LICENSE.
+-- todo.txt https://github.com/too-much-todotxt/spec LPeg lexer.
+
+local lexer = lexer
+local P, S = lpeg.P, lpeg.S
+
+local lex = lexer.new(...)
+
+local not_whitespace = lexer.any - lexer.space - P(':')
+local not_whitespace_word = not_whitespace^1
+
+
+-- Done/Complete items, map to comment style
+lex:add_rule('done', lex:tag(lexer.COMMENT, lexer.starts_line(lexer.to_eol('x '))))
+
+-- Priority
+--lex:add_rule('priority', lex:tag(lexer.LIST .. '.priority', lexer.starts_line(P('(') * lexer.upper * P(') '))))
+--lex:add_rule('priority', lex:tag(lexer.LIST, lexer.starts_line(P('(') * lexer.upper * P(') '))))
+
+local priority_a = lex:tag(lexer.LIST .. '.priority.a', lexer.starts_line('(A) '))
+local priority_b = lex:tag(lexer.LIST .. '.priority.b', lexer.starts_line('(B) '))
+local priority_c = lex:tag(lexer.LIST .. '.priority.c', lexer.starts_line('(C) '))
+local priority_d = lex:tag(lexer.LIST .. '.priority.d', lexer.starts_line('(D) '))
+local priority_e = lex:tag(lexer.LIST .. '.priority.e', lexer.starts_line('(E) '))
+local priority_f = lex:tag(lexer.LIST .. '.priority.f', lexer.starts_line('(F) '))
```

Comment:
You can actually write a loop (untested):

```
local priority = P(false)
for letter in string.gmatch('abcdefghijklmnopqrstuvwxyz', '.') do
  local tag = lex:tag(lexer.LIST .. '.priority.' .. letter, lexer.starts_line('(' .. letter:upper() .. ') '))
  priority = priority + tag
end
lex:add_rule('priority, priority)
```

Reply:
I feel a bit dumb, I wrote a script to generate the previous code 😆 . I keep forgetting this isn't a traditional parser, I can write code not just declariations! Thank you